package.json

```diff
@@ -101,6 +101,6 @@
     "yargs": "^16.0.3"
   },
   "engines": {
-    "node": "^12.18.3"
+    "node": "^14.15.4"
```

Comment:
Is it possible to have two supported options here for a while?

Reply:
Doesn't look like it, but also looks like it should be ok to leave it on 12.18 but run locally (may get warnings, but we don't have `engine-strict` turned on

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

Reply:
Oh, I was wrong - we do have `engine-strict` turned on! I guess that's why `yarn` fails on v14

![image](https://user-images.githubusercontent.com/1274422/108641738-d5f7b880-7505-11eb-8f20-707559182ce9.png)

Reply:
Updated to have required node version be 12 or 14